app/utils/index.ts

```diff
@@ -0,0 +1,39 @@
+import BigNumber from 'bignumber.js'
+
+export const shannonToCkb = (value: BigNumber | string | number): string => {
```

Comment:
It can be done by `parseUDTAmount`, and the name `parseUDTAmount` limits its usage